### PR TITLE
Bugfix/da 3300

### PR DESCRIPTION
**Pull request by lfpratik**

### CODE REVIEW

File: docker-images/base/patch/perceval-trigger-error-event.py.diff [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
++
++
++def get_smtp_handler():
++    return SDSSMTPHandler(
++        mailhost=('smtp.gmail.com', 465),
++        fromaddr='dev.analytics.error.bot@gmail.com',
++        toaddrs=['pratikk@proximabiz.com', 'sgupta@linuxfoundation.org',
++                 'fghiasy@linuxfoundation.org'],
++        subject='SDS Error - log',
++        credentials=('dev.analytics.error.bot@gmail.com', 'Mobile@311!'),
```
Comment (lukaszgryglicki) [inline docker-images/base/patch/perceval-trigger-error-event.py.diff:273]: Remove sensitive data from a public repo.

File: sources/cmd/syncdatasources/syncdatasources.go [hunk 2]
```diff
@@ -39,6 +39,7 @@ var (
 	gHint             int
 	noDropPattern     = regexp.MustCompile(`^(.+-f-.+|.+-earned_media|.+-dads-.+|.+-slack|.+-da-ds-gha-.+|.+-social_media|.+-last-action-date-cache)$`)
 	notMissingPattern = regexp.MustCompile(`^(.+-github-pull_request.*|.+-github-issue-raw.*|.+-github-repository-raw.*)$`)
+	emailRegex        = regexp.MustCompile("^[a-zA-Z0-9.!#$%&'*+\\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$")
```
Comment (lukaszgryglicki) [inline sources/cmd/syncdatasources/syncdatasources.go:42]: Not related to this PR?

File: sources/cmd/syncdatasources/syncdatasources.go [hunk 3]
```diff
@@ -4575,6 +4576,14 @@ func massageEndpoint(endpoint string, ds string, dads bool, idxSlug string, proj
...
 			e = append(e, endpoint)
 		}
+	} else if ds == lib.GoogleGroups {
+		// google groups endpoint should be a valid email address.
+		if len(endpoint) < 3 || len(endpoint) > 254 {
+			return
+		}
+		if ok := emailRegex.MatchString(endpoint); ok {
+			e = append(e, endpoint)
+		}
```
Comment (lukaszgryglicki) [inline sources/cmd/syncdatasources/syncdatasources.go:4586]: Not related to this PR?

**Review by lukaszgryglicki — CHANGES_REQUESTED** [review]
Remove the sensitive data from a public repo. I see changes not related to this PR.

File: docker-images/base/patch/perceval-trigger-error-event.py.diff (hunk 1 above)
Comment (lfpratik) [reply docker-images/base/patch/perceval-trigger-error-event.py.diff]: @lukaszgryglicki, can review pr once more time.

File: sources/cmd/syncdatasources/syncdatasources.go (hunk 2 above)
Comment (lfpratik) [reply sources/cmd/syncdatasources/syncdatasources.go:42]: I am not sure about this.

File: sources/cmd/syncdatasources/syncdatasources.go (hunk 3 above)
Comment (lfpratik) [reply sources/cmd/syncdatasources/syncdatasources.go:4586]: I am not sure about this.

**Pull request closed without merging**